GOVERNANCE.md

```diff
@@ -0,0 +1,130 @@
+# Release Working Group
+
+The Node.js Release Working Group (WG) maintains oversight
+over the Node.js Release and Long Term Support (LTS) teams. It
+manages the release and long term support shedule and policies
+for all Node.js releases.
+
+The WG has final authority over Releases including:
+
+* Release process and tools.
+* Content for all releases.
```

Comment:
This puts the Release Working Group above the CTC. If it is the Release WG that can decide what composes a release, they can decide (autonomously) to pull a semver-major commit they do not agree with from a semver-major release. In similar fashion, they can decide to put something in that is not present in nodejs/node.

All of this is very hypothetical, but hypothetical things happened before so I would like to avoid conflicts before they happen.

I do agree that the current model is the right model and they should be able to omit things, but this sentence is too broad.

How about:

```
Content for all releases, according to the contributions in nodejs/node and semver levels.
```

I hope it clarifies what I mean.